docs/webapp/applications/apps_aws_autoscaler.md

```diff
@@ -147,7 +147,8 @@ auto_scaler.v1.aws {
 }
 ```
 
-To configure a specific instance(s), add a regular expression to match the autoscaler's `Workers Prefix` under 
+#### Configure Specific Autoscaler Instances
+To configure a specific instance(s) in the Configuration Vault, add a regular expression to match the autoscaler's `Workers Prefix` under 
```

Comment:
@pollfly "To configure... in the configuration vault" -> "Use the configuration vault to configure specific instances: Add a regular..."